test/afterburner/CMakeLists.txt

```diff
@@ -0,0 +1,200 @@
+project(afterburner NONE)
+
+set(afterburner_sims "")
+set(afterburner_analysis "")
+set(afterburner_plots "")
+
+file(MAKE_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}/Plots)
+function(afterburner_test
+         test_name   # name of folder
+    )
+    message(STATUS "Configuring afterburner/${test_name}.")
+    set(test_folder ${CMAKE_CURRENT_BINARY_DIR}/${test_name})
+    file(MAKE_DIRECTORY ${test_folder})
+    file(MAKE_DIRECTORY ${test_folder}/Merged_spectra)
+    #file(MAKE_DIRECTORY ${test_folder}/Plots)
+
+
+    set(smash_afterburner_config "${CMAKE_CURRENT_SOURCE_DIR}/afterburner.yaml")
+
+    execute_process (
+    COMMAND bash -c "cd ${CMAKE_CURRENT_SOURCE_DIR}/${test_name} && unzip -o sampling0.zip"
```

Comment:
Since this is a single compressed file and not an archive, I am wondering it it would be better to use gzip...
Currently in the Virgo cluster we are using containers, so we can put inside what we want, moreover `zip` usually is very common elsewhere, however it is not installed in the "bare metal" OS:
```
g@uhuru:~/SMASH-dev/smash-analysis/test/afterburner$ ssh -J lxpool ginghir@virgo.hpc.gsi.de
X11 forwarding request failed on channel 0
Virgo Slurm Cluster – Job Submit Node
ginghir@lxbk0599:~$ zip -v
bash: zip: command not found
```
gzip at the highest compression level performs slightly better than standard zip:
```
g@uhuru:~/SMASH-dev/smash-analysis/test/afterburner/LHC$ gzip -9 sampling0
g@uhuru:~/SMASH-dev/smash-analysis/test/afterburner/LHC$ ls -l
total 175644
-rw-rw-r-- 1 g g 89696892 Dec 14 13:37 sampling0.gz
-rw-rw-r-- 1 g g 90156516 Mar 30 10:12 sampling0.zip
```
However, I think that it's unlikely that this will ever be an issue...

Reply:
I was not aware of this problem, thank you! I changed it to gzip